### DISCUSSION TITLE
How many hours does the magic link token expire?

### DISCUSSION THREAD

**Discussion by kazu-2020** [supabase/supabase#14167] [Questions] [answered]
I would like to know how many hours the Magic Link token is valid for.
Also, is it possible to change the expiration date of this token?? 🤔

**Comment by GaryAustin1** [comment, marked as answer]
24 hours default.

![image](https://user-images.githubusercontent.com/54564956/236626530-4efdee1a-d795-46dd-a4ac-877e46317008.png)

**Reply by max-programming** [reply]
Is there any way to set it to no expiry?

**Reply by GaryAustin1** [reply]
No.

**Comment by kazu-2020** [comment]
@GaryAustin1 
Thanks!
